### PR TITLE
Update manage-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about the time it takes to propagate the changes made to let external users email the group.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6612

File: microsoft-365/admin/create-groups/manage-groups.md [hunk 1]
```diff
@@ -76,6 +76,9 @@ This option is great if you want to have a company email address such as info@co
     
 4. Select **Save**.
 
+> [!NOTE]
+> The changes could take 30 minutes to propogate and let external users email the organization.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/manage-groups.md]:
```suggestion
> The changes could take up to 30 minutes to propagate and let external users email the organization.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**